.github/workflows/ci.yml

```
@@ -0,0 +1,39 @@
+name: CI for Integration Service FIWARE SystemHandle
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  fiware-sh_CI:
+    runs-on: ubuntu-20.04
+    container: ubuntu:focal
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: src/fiware-sh
+
+      - name: Download required dependencies
+        run: |
+          apt update
+          DEBIAN_FRONTEND=noninteractive apt install -y cmake gcc g++ git libboost-dev libboost-program-options-dev libyaml-cpp-dev libcurlpp-dev libasio-dev
+
+      - name: Install colcon
+        run: |
+          apt install -y python3-pip
+          pip3 install -U colcon-common-extensions
+
+      - name: Download the Integration Service
+        run: |
+          git clone -b feature/json-xtypes-subnode --recursive https://github.com/eProsima/Integration-Service src/integration-service
```

Change after https://github.com/eProsima/Integration-Service/pull/86 merged
```suggestion
          git clone --recursive https://github.com/eProsima/Integration-Service src/integration-service
```